docs/install/osx_setup.md

```diff
@@ -146,10 +157,11 @@ We have installed MXNet core library. Next, we will install MXNet interface pack
 To install the MXNet Python binding navigate to the root of the MXNet folder then run the following:
 
 ```bash
-$ cd python
-$ pip install -e .
+virtualenv -p`which python3` py3_venv
```

Comment:
Same for `virtualenv`. This has nothing to do with MXNet. We should let users to choose their own tool (such as conda) instead.

Reply:
Im afraid that if we don't specify either one users will override the system global installation of mxnet, it's good hygiene to scope it to a virtualenv.

Reply:
I think it's a general practice for users to install any package not related to mxnet. But this is non blocking to me. @aaronmarkham please advise.

Reply:
It depends where it's going. When we have a detailed guide, it is fine to add recommended practices as new users may not know these details like a seasoned Python developer might.
When we have a very succinct section like the install page, I'd leave out these details.
So, I'd leave this in here since osx_setup.md is meant to be a detailed guide.

I do suggest another name though.  Like `mxnet_p36`, which is used as the names of the Conda environments on the DLAMI. Speaking of Conda, it would be good to mention this as an option too. It's ok to not document those steps as you're providing the virtualenv steps already. It's nice to let people know that mxnet will work fine in a Conda environment if that's what they want to do.

Reply:
renamed to mxnet_py3